src/ai/backend/manager/cli/image.py

```diff
@@ -110,3 +115,36 @@ def alias(cli_ctx, alias, target, architecture) -> None:
 def dealias(cli_ctx, alias) -> None:
     """Remove an alias."""
     asyncio.run(dealias_impl(cli_ctx, alias))
+
+
+@cli.command()
+@click.argument("alias")
+@click.pass_obj
+def validate_image_alias(cli_ctx: CLIContext, alias: str) -> None:
+    """Validate a local/remote image's labels and platform tag formats by image alias."""
+    try:
+        result = asyncio.run(validate_image_alias_impl(cli_ctx, alias))
+        for key, value in result.items():
+            print(f"{key[11:]:<20}: ", end="")
+            print(*value) if isinstance(value, list) else print(value)
+    except Exception:
+        sys.exit(ExitCode.FAILURE)
```

Comment:
I think it is better to just raise appropriate Click exceptions instead of manually catching it here.
(ref: https://click.palletsprojects.com/en/8.1.x/api/#exceptions)